app/forms/claimant_form.rb

```diff
@@ -45,9 +45,9 @@ class ClaimantForm < Form
   validates :address_country, inclusion: { in: COUNTRIES }
   validates :fax_number,    presence: { if: :contact_preference_fax? }
   validates :email_address, presence: { if: :contact_preference_email? },
-                            email: { if: :contact_preference_email? },
                             ccd_email: { if: :contact_preference_email? },
                             length: { maximum: EMAIL_ADDRESS_LENGTH }
+  validates :email_address, email: { mode: :strict }, if: :contact_preference_email?
```

Comment:
Why did this need its own line rather than adding mode: :strict into the existing email: {} block ?